site2/docs/administration-load-balance.md

```diff
@@ -6,96 +6,79 @@ sidebar_label: Load balance
 
 ## Load balance across Pulsar brokers
 
-Pulsar is an horizontally scalable messaging system, so it is a core requirement that the traffic
-in a logical cluster must be spread across all the available Pulsar brokers, as evenly as possible.
+Pulsar is an horizontally scalable messaging system, so the traffic
+in a logical cluster must be spread across all the available Pulsar brokers as evenly as possible, which is a core requirement.
 
-In most cases, this is true out of the box and one shouldn't worry about it. There are, though,
-multiple settings and tools to control the traffic distribution and they require a bit of
-context to understand how the traffic is managed in Pulsar.
+In most cases, this requirement is true out of the box and you should not worry about it. Though,you can use multiple settings and tools to control the traffic distribution and they require a bit of context to understand how the traffic is managed in Pulsar.
```

Comment:
refine this sentence, pay attention to punctuation and try to make it easy to read and understand.